VATSpy.dat

```diff
@@ -18156,8 +18156,8 @@ LZBB|Bratislava||LZBB
 MDCS|Santo Domingo||MDCS
 MHCC|Cenamer||MHCC
 MKJK|Kingston||MKJK
-MMEX|Mexico|MEX|MMEX
-MMFR|Mexico|MEX|MMFR
+MMEX|Mexico|MMEX
+MMFR|Mexico|MMFR

```

Comment:
You still need two ||

So should be like this:
MMEX|Mexico||MMEX
MMFR|Mexico||MMFR

Reply:
Ofc, my bad! Fixed on 24764837e59e5d759fcf0c05f175b5ba76226f8b